### ISSUE TITLE
Dessert vs Confection

### ISSUE THREAD

**Issue by cmrn-rhi** [FoodOntology/foodon#222]
## Dessert vs Confection

There isn't a clear delineation between Dessert and Confection in FoodOn.

### **dessert food**
[FOODON:03303220](http://purl.obolibrary.org/obo/FOODON_03303220)

definition: _Missing_
comment: LanguaL curation note: Use for supplements based on plants, yeast, algae, and fungi (index in facet B). If the supplement also contains vitamins or minerals use the appropriate descriptor under * DIETARY SUPPLEMENT, COMBINATION*

### **confectionery food product** 
[FOODON:00001149](http://purl.obolibrary.org/obo/FOODON_00001149)

definition: Food items that are rich in sugar, any one or type of which is called a confection. Modern usage may include substances rich in artificial sweeteners as well.

---

It seems like the key difference is that a dessert is served as a last course of a meal (https://wikidiff.com/confection/dessert). I think it still works fine to embody other dessert classes, but I don't see the sense in having "dessert food" home items like "baklava" as "baklava" can be consumed anytime (as a confectionary) and only becomes a dessert when consumed as a specific time.

![image](https://user-images.githubusercontent.com/48695054/185206250-79698bb7-e5ff-4ae9-8536-89d361bd934a.png)

---
### New Definitions
**dessert food:** A food product served as a course that concludes a meal.
definition source: https://en.wikipedia.org/wiki/Dessert
comment: This term can apply to confections, fruits, and even savory foods.

**confectionary food product:** A food product which is rich in sugar and carbohydrates.
definition source: https://en.wikipedia.org/wiki/Confectionery
comment: Modern usage may include substances rich in artificial sweeteners as well.

### Rehome Subclasses
Dessert food subclasses that aren't a dessert specific class should be rehomed under "confectionary food product".

**Comment by maweber-bia** [comment]
I think it would be useful to specify that a dessert food is a **sweet** course  that concludes a meal
if you read the wikipedia page, it is mentioning "Some cultures sweeten foods that are more commonly savory to create desserts." There is no savoury product as such.

I think we may made a more specific definition for confectionary food products:

 Defintion: A confectionary food product is a food product which is highly sweet with sugars or artificial sweeteners.
Comment : This term can apply to sugar confectionary like candies, to chocolate confectionary, and to sweet baked products

**Comment by cmrn-rhi** [comment]
> I think it would be useful to specify that a dessert food is a sweet confection that concludes a meal

I agree that the more common definition is to go for sweet food, but I was leaning on being more descriptive than prescriptive as I generally wouldn't consider a cheese platter as necessarily sweet but have been served several as "desserts" in home life and restaurants. Perhaps I should've linked the Merriam-Webster definition that is a bit more align with this perspective "a usually sweet course or dish (as of pastry or ice cream) usually served at the end of a meal" (https://www.merriam-webster.com/dictionary/dessert). That being said this isn't a hill I want to die on ;)

Your recommended changes to "confectionary food product" sound good to me, although I would lean towards dropping the "is a food product" to simplify to referencing only one parent term level.
> Defintion: A confectionary food product ~~is a food product~~ which is highly sweet with sugars or artificial sweeteners.

**Comment by maweber-bia** [comment]
of course you are right, definition for a confectionary food product should read "A food product which is highly sweet with sugars or artificial sweetener"

As for dessert, in France it is "dessert" or "cheese" at the end of the meal, or even "cheese" and then "dessert"!
But I can agree with the Merriam-Webster defintion, except that I would not have repeat "ususally" twice ;-)

this means : " "a usually sweet course or dish (as of pastry or ice cream) served at the end of a meal"

**Comment by cmrn-rhi** [comment]
**FoodOn Curation Meeting Discussion Notes (2022/10/13):**
Need to make sure the labels clearly represents what is being described. E.g., "dessert food: A food product served as a course that concludes a meal." is talking about a meal/food course so we should try to capture it in the label to avoid ambiguity. That way people wanting to use it categorically (perhaps as a synonym for "confectionary food product") to describe sweet foods consumed at any point of day don't mix them up.
We are going to discuss meals/courses next curation meeting (based on work done during the ICBO workshop); after we have clearly defined that micromodel it should hopefully be easier to resolve this issue.

**Comment by ddooley** [comment]
On a related note, some of this touches on meal type discussion on: Meal types and meal courses #236

**Comment by maweber-bia** [comment]
For confectionary see #235
